includes/polling/applications.inc.php

```diff
@@ -1,13 +1,22 @@
 <?php
 
+use App\Models\Application;
+
 $app_rows = dbFetchRows('SELECT * FROM `applications` WHERE `device_id`  = ?', [$device['device_id']]);
 
 if (count($app_rows) > 0) {
     foreach ($app_rows as $app) {
         echo 'Application: ' . $app['app_type'] . ', app_id=' . $app['app_id'];
         $app_include = \LibreNMS\Config::get('install_dir') . '/includes/polling/applications/' . $app['app_type'] . '.inc.php';
         if (is_file($app_include)) {
+            $app_model = Application::find($app['app_id']);
```

Comment:
Remember that Application::find() is actually a database query.  So you should fetch that outside the loop.  And try to only call it once in general.